### PR TITLE
doma: switch to u-boot-android-bazel only builds

### PR DESCRIPTION
**Pull request by arminn**
This patch aims to remove the outdated u-boot-android and
replace it with the u-boot-android-bazel recipe for building
u-boot on the doma platform. The switch is necessary to
leverage the latest features required for Android boot.
Please note that this change will transition to using the
current Google Android u-boot with specific commits frozen
in the manifest. However, the standard Google Android u-boot
relies on persistent storage for bootconfig, while all supported
doma variants utilize bootconfig without storing it in persistent
storage. Consequently, an additional patch will be applied
to disable the mentioned feature.

### CODE REVIEW

**Comment by arminn** [comment]
Under testing ...

**Comment by arminn** [comment]
@svlad-90 plz review

**Comment by svlad-90** [comment]
Hi @arminn,

 Reviewed-by: Goncharuk Vladyslav <goncharuk_vladyslav@epam.com>

File: meta-xt-control-domain/recipes-guest/doma/files/0001-xenvm-do-not-use-persistent-storage-for-bootconfig.patch
Comment (lorc) [inline meta-xt-control-domain/recipes-guest/doma/files/0001-xenvm-do-not-use-persistent-storage-for-bootconfig.patch:1]: Could you please move changes  related to this patch file into a separate commit? As I understand, it has nothing to do with the switching process per se.

Comment (lorc) [reply meta-xt-control-domain/recipes-guest/doma/files/0001-xenvm-do-not-use-persistent-storage-for-bootconfig.patch:1]: Okay, I see. Could you please mention this change in the commit message? RN this change is not justified in any way.

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by arminn** [comment]
rb added

**Pull request merged**